test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.anno.json

```diff
@@ -15,15 +15,10 @@
             "class": "sifive.enterprise.grandcentral.AugmentedGroundType",
             "ref": {
               "circuit": "Top",
-              "module": "DUT",
+              "module": "MyView_companion",
               "path": [],
-              "ref": "w",
-              "component": [
-                {
-                  "class": "firrtl.annotations.TargetToken$Field",
-                  "value": "uint"
-                }
-              ]
+              "ref": "uint",
+              "component": []
```

Comment:
This is a change to an input annotation. Is this intended and does the old formulation work? I was expecting to see no changes like this to tests.

Reply:
This change was not required, removed it.